packet_spec.py

```diff
@@ -21,6 +23,38 @@ class TelemetryPacketSubType(Enum):
     ARMING_STATE = 3
     ACT_STATE = 4
     WARNING = 5
+    ACT_REQ = 6
+    ACT_ACK = 7
+    ARM_REQ = 8
+    ARM_ACK = 9
```

Comment:
These subpacket types aren't for telemetry but are for control and should probably go in their own enum for consistency. You can refer to page 4 of the comm format
https://github.com/CarletonURocketry/hybrid-comm-format/blob/gh-pages/spec.pdf